Software/Python/Examples/easyMotors.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+#
+# https://www.dexterindustries.com/GoPiGo/
+# https://github.com/DexterInd/GoPiGo3
+#
+# Copyright (c) 2017 Dexter Industries
+# Released under the MIT license
+# (http://choosealicense.com/licenses/mit/).
+#
+# For more information see
+# https://github.com/DexterInd/GoPiGo3/blob/master/LICENSE.md
+#
+# This code is an example for controlling the GoPiGo3 motors.  This uses
+# the EasyGoPiGo3 library.  You can find more information on the library
+# here:  http://gopigo3.readthedocs.io/en/latest/api-basic.html#easygopigo3
+#
+# Results:  The GoPiGo3 will move forward for 2 seconds, and then
+# backward for 2 second.
+
+
+# import the time library for the sleep function
+import time
+# import the GoPiGo3 drivers
+import easygopigo3 as easy
+
+# Create an instance of the GoPiGo3 class.
+# GPG will be the GoPiGo3 object.
+gpg = easy.EasyGoPiGo3()
+
+gpg.forward()
+
+time.sleep(2)
```

Comment:
Ironically I could not get drive_cm to work after calling gpg.forward().  Any tips?